lib/server/ServerDispatcher.js

```diff
@@ -41,6 +38,20 @@ var ServerDispatcher = {
 
 };
 
+function findHandlerForRoute(fragment) {
+    var handler;
+
+    for (var i = 0, len = handlers.length; i < len; i++) {
+        handler = handlers[i];
+
+        if (!handler.route.test(fragment)) {
```

Comment:
sure. i had it inverted the other way first but it wasn't as pretty. i'll switch it back